Cargo.toml

```diff
@@ -427,6 +427,7 @@ pulsar-integration-tests = ["sinks-pulsar"]
 splunk-integration-tests = ["sinks-splunk_hec", "warp"]
 
 shutdown-tests = ["sources","sinks-console","sinks-prometheus","sinks-blackhole","unix","rdkafka","transforms-log_to_metric","transforms-lua"]
+cli-tests = []
```

Comment:
I think this is fine as a default test? I'm not sure we need to add another feature for this quite yet.

Reply:
Fair enough.